.pre-commit-config.yaml

```diff
@@ -0,0 +1,38 @@
+# Auto formats python code
+repos:
+
+-    repo: https://github.com/psf/black.git
+     rev: 23.1a1
+     hooks: 
+     - id: black
+
+
+-    repo: https://github.com/pre-commit/pre-commit-hooks
+     rev: v4.4.0
+     hooks:
+     - id: check-added-large-files
+     - id: check-ast
+     - id: check-builtin-literals
+     - id: check-byte-order-marker
+     - id: check-case-conflict
+     - id: check-docstring-first
+     - id: check-executables-have-shebangs
+     - id: check-json
+     - id: check-merge-conflict
+     - id: check-symlinks
+     - id: check-toml
+     - id: check-vcs-permalinks
+     - id: check-yaml
+     - id: debug-statements
+     - id: destroyed-symlinks
+     - id: detect-private-key
+     - id: double-quote-string-fixer
```

Comment:
Wait! Get rid of `double-quote-string-fixer` please, I thought it did the opposite and I want my double quotes intact!